packages/core/src/components/badge/badge.scss

```diff
@@ -0,0 +1,28 @@
+:host {
+  display: inline-block;
+}
+
+.atom-badge {
+  border-radius: var(--spacing-base);
+  color: var(--color-neutral-white);
+  font-size: var(--text-badge);
+  letter-spacing: var(--text-badge-letter);
+  padding: var(--spacing-xxsmall) var(--spacing-xsmall);
+
+  &[color='warning'] {
+    background-color: var(--color-contextual-warning-regular);
+  }
+
+  &[color='danger'] {
+    background-color: var(--color-contextual-error-regular);
+  }
+
+  &[color='light'] {
```

Comment:
Question: Is this a "disabled" state? I'm thinking if the **light** is the best work for this.

Reply:
We cannot have a type of disabled for this component. Because badges are different from tags. Badges have no interaction, it's just visual and static that doesn't interact with anything, it wouldn't make sense to have a type of disabled. In the tag component it would make sense.

[More infos](https://www.delldesignsystem.com/components/badge/)

Reply:
In Figma is a disabled property for this badge

![image](https://github.com/juntossomosmais/atomium/assets/75763403/1ca34c36-aec3-4827-a9aa-14315367c306)

Reply:
So does it make sense to have a type `disabled` instead of `light?` As it's something that doesn't have any action, it would be interesting to pass light to symbolize the color, but for me it whatever. 

What do you think? @juntossomosmais/loyalty-frontends

Reply:
I also would like to have more opinions about it. Both seem strange to me now, but **disabled** at least fits with the Figma layout. @JulioCVaz

Reply:
This realy does make me curious, maybe the usage of disable was just to "keep consistency", but once the component is no interactible, this does not make sense 🤔

Reply:
Yeah, totally. How about **neutral** (--color-**neutral**-light-3)?

Reply:
I hit the light key because it is the ''standard'' used for that color in the project.

Atomium use:
![image](https://github.com/juntossomosmais/atomium/assets/59345698/08a5228a-85ed-488d-9ceb-6b4d16a55f1d)

But component Alert use 'neutral' 👀
![image](https://github.com/juntossomosmais/atomium/assets/59345698/d97b0b8e-5c7b-476b-9119-b0b5e49498f7)

[In addition to also being used in ionic](https://ionicframework.com/docs/theming/colors):
![image](https://github.com/juntossomosmais/atomium/assets/59345698/9ca52abf-a5ce-4463-aaba-ceac0c7718d8)

And disabled only use in interactions with states

Reply:
Guess the name of the color itself is "neutral" and "light" is a variation, don't you think?

Reply:
Looking at this whole thread, I honestly think type="neutral" is the way to go, as is `Alert`. So we keep a default for those components that are not interactive and use that variation. @tassioFront @gabrielduete @schirrel

Reply:
I agree to use `neutral`!

Thanks for the cooperation, guys!

Made: https://github.com/juntossomosmais/atomium/pull/239/commits/99f9e2b47d596791cc4f7841cdee07336f6d8ba0